build.xml

```diff
@@ -119,10 +120,11 @@
     </target>
 
     <!-- TEST -->
-    <target name="test" depends="compile, set_excluded_test_groups" description="Run unit tests">
+    <target name="run-test" depends="compile, set_excluded_test_groups" description="Run unit tests">
```

Comment:
I think you might want to turn `run-test` into a macro using `macrodef` (there are examples of this elsewhere in this build file). Macros can be parameterized a bit more cleanly (without using `antcall`).